packages/theming/src/elements/theme/index.ts

```diff
@@ -112,7 +209,7 @@ const lineHeights = {
 const palette = { ...PALETTE };
 
 /* Exclude product palette from the theme */
-delete palette.product;
+delete (palette as any).product;
```

Comment:
I think another way around this without using `any` is...

```ts
const { ...palette, product } = PALETTE;
```

Reply:
Since `PALETTE` is inferring its type this can cause a bunch of `Index signature is missing in type` errors. The original `...` operator is what keeps this working.

We probably shouldn't be inferring those types anyway, but that is probably a larger scope change than we should do here.